cli/Makefile

```diff
@@ -46,6 +46,13 @@ ci-copy-binary:
 	cp $(OUTPUT_DIR)/$(VERSION)/cli-hasura-* /build/_cli_output/binaries
 	echo "$(VERSION)" > /build/_cli_output/version.txt
 
+.PHONY: ci-copy-cli-ext
+ci-copy-cli-ext:
+	cp -r /build/_cli_ext_output ./cliext/bin
+
+copy-cli-ext:
+	cp -r ../cli-ext/bin ./cliext/bin
```

Comment:
we will probably merge these into one and choose the source path based on the `CI` variable